src/models/TransactionEvent.class.js

```diff
@@ -23,8 +23,14 @@ class TransactionEvent {
     this.updated_friendly = dates.utcToDisplay(this.updated)
     this.amount_friendly = penceToPoundsWithCurrency(this.amount)
     const transactionType = this.type.toLowerCase()
-    if (this.amount && (transactionType === 'refund' || transactionType === 'dispute')) {
-      this.amount_friendly = `–${this.amount_friendly}`
+    if (this.amount) {
+      if (transactionType === 'refund') {
+        this.amount_friendly = `–${this.amount_friendly}`
+      } else if (transactionType === 'dispute') {
+        if (this.state.status !== 'won') {
```

Comment:
This nested if statement could be simplified:
`else if (transactionType === 'dispute' && this.state.status !== 'won') {
   this.amount_friendly = `–${this.amount_friendly}`
}`